feat(router): Stub new components and connect with vue-router

astra137 opened the pull request:
Adds vue-router, stubs new components, and tries to mimick the wireframe. Routing is using params to
select a game and element to edit.

#14

Commit:
feat(router): Stub new components and connect with vue-router

Adds vue-router, stubs new components, and tries to mimick the wireframe. Routing is using params to
select a game and element to edit.

#14

Commit:
build(dependencies): Add vue-router as dependency

monochromicon commented:
It looks pretty good to me, but can we convert the game IDs in the router to unique slugs?

maccelerated commented:
Sure, I'll change the example to match.

Commit:
fix(router): Move game dropdown, simplify element list, prototype element slugs

Adds prototype element slugs that follow game slug and cleans interface a bit.

#8

puregarlic reviewed (approved):
Looks good to me.
